esp/src/eclwatch/QuerySetDetailsWidget.js

```diff
@@ -232,7 +232,7 @@ define([
                         tooltip += " " + newValue[i].Time;
                 }
                 this.graphsTab.set("tooltip", tooltip);
-            } else if (name === "ResourceURLCount" && newValue) {
+            } else if (name === "ResourceURLCount" && newValue - 1) {
                 this.widget._Resources.set("title", this.i18n.Resources + " (" + newValue + ")");
```

Comment:
But, doesn't the else if prevent execution if newValue is anything but a positive integer?